src/main/java/com/swp/common/domain/users.java

```diff
@@ -0,0 +1,27 @@
+package com.swp.common.domain;
+
+import org.springframework.data.annotation.CreatedDate;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+public class users {
```

Comment:
Java에서 클래스명은 대문자로 하는게 컨벤션이에요!

클래스명을 users에서 `User` 로 바꾸는게 어떨까요?
```suggestion
@Entity
@Table(name = "users")
public class User {
```

Reply:
users(복수형)에서 user(단수형)로 한 이유는 이후에

```java
Users users = new Users(..);
```

처럼 코드가 나올텐데, 그럼 users 변수가 *사용자들* 인지, 아니면 *사용자* 인지 헷갈릴 수 있기 때문이에요.